src/client/src/theme/ThemeContext.tsx

```diff
@@ -73,7 +73,7 @@ export const ThemeProvider: React.FC<Props> = ({
   )
 }
 
-export const useTheme = () => {
+export const useThemeName = () => {
```

Comment:
What was your reasoning behind this name change? The hook returns more than just the theme name.

Reply:
Noted in commit message of - https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/2063/commits/9baeda8c1ff384456174c5cbfbee23275a159e01 - to avoid confusion with
https://styled-components.com/docs/api#usetheme which returns a theme object and is used elsewhere in the app, e.g. https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/client/src/OpenFin/Window/LayoutLock.tsx#L3

Open to suggestions for a better name, or we can just drop this commit if the original is preferred.

Reply:
It doesn't bother me too much, I think we probably have other instances of similar names with dependencies.
If you feel strongly about it then maybe something like `useApplicationTheme` or `useReactiveTraderTheme`, or drop the change.